addons/binding/org.openhab.binding.kostalpikoiqplenticore/README.md

```diff
@@ -0,0 +1,2964 @@
+Number:Energy                MyPikoIq85_DEVICE_LOCAL_HOMECONSUMPTION_TOTAL            <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_HOMECONSUMPTION_TOTAL"}
+Number:Energy                MyPikoIq85_DEVICE_LOCAL_LIMIT_EVU_ABSOLUTE               <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_LIMIT_EVU_ABSOLUTE"}
+Number:Dimensionless         MyPikoIq85_DEVICE_LOCAL_LIMIT_EVU_RELATIV                <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_LIMIT_EVU_RELATIV"}
+Number:Time                  MyPikoIq85_DEVICE_LOCAL_WORKTIME                         <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_WORKTIME"}
+Number:ElectricCurrent       MyPikoIq85_DEVICE_LOCAL_AC_PHASE_1_CURRENT_AMPERAGE      <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_AC_PHASE_1_CURRENT_AMPERAGE"}
+Number:Energy                MyPikoIq85_DEVICE_LOCAL_AC_PHASE_1_CURRENT_POWER         <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_AC_PHASE_1_CURRENT_POWER"}
+Number:ElectricPotential     MyPikoIq85_DEVICE_LOCAL_AC_PHASE_1_CURRENT_VOLTAGE       <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_AC_PHASE_1_CURRENT_VOLTAGE"}
+Number:ElectricCurrent       MyPikoIq85_DEVICE_LOCAL_AC_PHASE_2_CURRENT_AMPERAGE      <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_AC_PHASE_2_CURRENT_AMPERAGE"}
+Number:Energy                MyPikoIq85_DEVICE_LOCAL_AC_PHASE_2_CURRENT_POWER         <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_AC_PHASE_2_CURRENT_POWER"}
+Number:ElectricPotential     MyPikoIq85_DEVICE_LOCAL_AC_PHASE_2_CURRENT_VOLTAGE       <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_AC_PHASE_2_CURRENT_VOLTAGE"}
+Number:ElectricCurrent       MyPikoIq85_DEVICE_LOCAL_AC_PHASE_3_CURRENT_AMPERAGE      <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_AC_PHASE_3_CURRENT_AMPERAGE"}
+Number:Energy                MyPikoIq85_DEVICE_LOCAL_AC_PHASE_3_CURRENT_POWER         <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_AC_PHASE_3_CURRENT_POWER"}
+Number:ElectricPotential     MyPikoIq85_DEVICE_LOCAL_AC_PHASE_3_CURRENT_VOLTAGE       <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_AC_PHASE_3_CURRENT_VOLTAGE"}
+Number:Energy                MyPikoIq85_DEVICE_LOCAL_AC_CURRENT_POWER                 <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_AC_CURRENT_POWER"}
+Number:ElectricCurrent       MyPikoIq85_DEVICE_LOCAL_PVSTRING_1_AMPERAGE              <energy> { channel="kostalpikoiqplenticore:PIKOIQ85:MyPikoIq85:DEVICE_LOCAL_PVSTRING_1_AMPERAGE"}
+
```

Comment:
It's a very extensive readme with ~3000 lines of code. :+1:  Maybe it's a bit too extensive. :wink:

Reply:
Suggestion:
Downsize the readme to one device of each class (1x PIKO IQ, 1x PlentiCore with battery, 1x PlentiCore without battery)

Is that an acceptable compromise? :)

Reply:
If these are very similar devices that would be the better approach! I haven't looked into detail at all the thing types yet. Fewer lines of code also attracts more reviewers. :-)

I do see you've made great progress with updating the license and removing all the JARs. :smile: :+1:

Reply:
The file is downsized as described above